deploy/olm-catalog/tektoncd-pipeline-operator/0.3.1/tektoncd-pipeline-operator.v0.3.1.clusterserviceversion.yaml

```diff
@@ -43,8 +64,8 @@ spec:
       - description: Version of Tektoncd Pipeline installed
         displayName: Tektoncd Pipeline Version
         path: version
-  description: Tektoncd Operator Cluster Service Version
-  displayName: Tektoncd Operator
+  description: Kubernetes operator to install TektonCD-Pipeline
+  displayName: TektonCD Pipeline Operator
```

Comment:
I think most if not all references to "TektonCD Pipeline" should be "Openshift pipeline" for the downstream operator.

Reply:
+1